apps/desktop-wallet/src/pages/NewWallet/CreateWalletPage.tsx

```diff
@@ -62,7 +77,8 @@ const CreateWalletPage = ({ isRestoring = false }: { isRestoring?: boolean }) =>
         passwordError = t`Insecure password`
```

Comment:
Nit: not part of this PR but signalling because I'm seeing it :)  

```ts
t('Password is too weak')
t('Insecure password')
```

Reply:
Fixed in 153e081b1 in PR #504